actor-tests/src/test/java-jdk21-only/org/apache/pekko/actor/StashJavaAPITestUntypedActors.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
```

Comment:
should this not have the normal Apache header - was it really copied from Akka like this header claims?

Reply:
I adjusted some code from it, maybe that's fine?